students/lakatorz_izaak/lesson_09_reading_and_writing_files/mad_libs.py

```diff
@@ -0,0 +1,22 @@
+
+text_file = open('../lesson_09_reading_and_writing_files/file.txt')
+
+text_to_work = text_file.read()
+
+# text_to_work = text_to_work.split()
+
+print(text_to_work)
+
+adj = input('Enter an adjective: ')  # silly
+noun = input('Enter a noun: ')       # chandelier
+verb = input('Enter a verb: ')       # screamed
+adv = input('Enter an adverb: ')     # quickly
+
+text_to_work = text_to_work.replace('ADJECTIVE', str(adj))
+text_to_work = text_to_work.replace('NOUN', noun)
+text_to_work = text_to_work.replace('VERB', verb)
+text_to_work = text_to_work.replace('ADVERB', adv)
```

Comment:
To jest ok, alternatywnie możesz zrobić łańcuch wywołań i użyć backslasha jako kontynuacji linii. Wtedy nie musisz powtarzać `text_to_work = text_to_work....` w każdej linii
```
text_to_work = text_to_work \
  .replace('ADJECTIVE', str(adj)) \
  .replace('NOUN', noun) \
  .replace('VERB', verb) \
  .replace('ADVERB', adv)
```